library/src/morpheus-master/sass/modules/grader/_base.scss

```diff
@@ -233,12 +233,30 @@ sakai-grader {
       }
     }
 
-    sakai-file-picker {
-      margin-left: 10px;
+    sakai-grader-file-picker {
 
       .sakai-file-picker-list {
         margin-top: 5px;
```

Comment:
I'd prefer if margins/paddings used `calc(#{$standard-spacing} RATIO)` to keep things evenly spaced everywhere. $standard-spacing defaults to 16px, so divide by 2,3,4 to get what you need.

`calc(#{$standard-spacing} / 2)` = 8px